store/cachekv/store_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
Please use the `require` package instead of `assert`

Reply:
when is it appropriate to use assert and when require? a bit confused on this

Reply:
https://github.com/stretchr/testify#require-package

The test case fails immediately on the first error.